src/scripts/Controller.js

```diff
@@ -1324,15 +1349,18 @@ export default class SitemapController {
 	}
 
 	addSelector() {
-		const parentSelectorId = this.state.currentParentSelectorId;
 		const sitemap = this.state.currentSitemap;
+		const parentSelectorId = sitemap.new_version
+			? this.state.currentParentSelectorId.uuid
+			: this.state.currentParentSelectorId.id;
```

Comment:
Empty sitemap has new_version = false, so first added selector will be unreachable until reload.